### PR TITLE
Some improvements for the docs

### PR DESCRIPTION
**Pull request by psychon**
So far this saves large parts of the "my first awesome"-page from the wiki and makes the index page that LDoc generates more useful.

### CODE REVIEW

File: docs/07-my-first-awesome.md [hunk 1]
```diff
@@ -34,46 +34,47 @@ between Ctrl and Alt. You can change the modkey if you want, but we'll get to
 that later. An `xterm` window will pop up. You can of course use your favourite
 terminal if you like. Again, more on that later. Now press *Mod4+Shift+c* to
 close the terminal. This is the command which lets you close any application.
-Awesome has a very elegant way to launch programs: press *Mod4+r*. Now you will
+Awesome has a elegant way to launch programs: press *Mod4+r*. Now you will
```
Comment (psychon) [inline docs/07-my-first-awesome.md]: an elegant?

File: docs/07-my-first-awesome.md [hunk 2]
```diff
@@ -163,7 +164,7 @@ yourself some trouble by using `Xephyr` to test your changes.
 
 Open a terminal and run the following command:
 
-    Xephyr :1 & sleep 1 ; DISPLAY=:1 awesome
+    Xephyr :5 & sleep 1 ; DISPLAY=:5 awesome
```
Comment (psychon) [inline docs/07-my-first-awesome.md]: What's so special about display 5?

File: docs/07-my-first-awesome.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+awesome step by step and finally come up with a working desktop, providing some
+basic knowledge on how to configure awesome in the process. We will start with
+the default `rc.lua` that comes with awesome, play around a little and try to
+make it more beautiful and useful with widgets. For this tutorial you don't need
+any programming skills besides using a text editor.
+
+## Explore awesome
+
+I will assume that you properly installed awesome through your distribution's
+package manager or compiled it from source. You should now add exec awesome to
```
Comment (psychon) [inline docs/07-my-first-awesome.md]: :+1:

**Comment by psychon** [comment]
Updated, squashed your fixup commit and added another new one of my one. The only remaining question is out of curiosity: Why is display 5 better than 1?

File: docs/07-my-first-awesome.md (hunk 1 above)
Comment (blueyed) [inline docs/07-my-first-awesome.md]: Yep.

**Comment by blueyed** [comment]
> Why is display 5 better than 1?

I've thought that :1 is more likely to be in use already.. (we're also using :5 for the tests)

**Pull request merged**